pages/home.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""Page model for home page."""
+
+from pypom import Page
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+
+class HomePage(Page):
+    """Page object model for home page."""
+
+    def wait_for_page_to_load(self):
```

Comment:
With PyPOM 2.0 we should avoid overriding `wait_for_page_load`. See http://pypom.readthedocs.io/en/latest/user_guide.html#upgrading-to-2-0 for details.